samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/evaluation/SubmissionNavBean.java

```diff
@@ -86,7 +83,8 @@ public void populate(List<AgentResults> agentResultsList, String currentGradingI
                     String displayName = agent.getLastName() + ", " + agent.getFirstName()
                         + " (" + agent.getAgentDisplayId()  + ")";
                     String optionLabel = displaySubmissionDate
-                            ? displayName + " - " + userTimeService.dateTimeFormat(agent.getSubmittedDate(), locale, DateFormat.MEDIUM)
+                            ? displayName + " - " + userTimeService.dateTimeFormat(agent.getSubmittedDate().toInstant(),
```

Comment:
Ummm, does this consider the users' locale?

Reply:
Yes, it will get it from the ResourceLoader as well internally.